script/add-payment-partials-to-marriage-abroad-services-data.rb

```diff
@@ -1,8 +1,124 @@
 countries = %w(
-  colombia
+  afghanistan
+  albania
+  algeria
+  american-samoa
+  andorra
+  angola
+  armenia
+  aruba
+  austria
+  azerbaijan
+  bahrain
+  benin
+  bhutan
+  bolivia
+  bonaire-st-eustatius-saba
+  bosnia-and-herzegovina
+  brazil
+  bulgaria
+  burkina-faso
+  burundi
+  cape-verde
+  central-african-republic
+  chad
+  chile
+  comoros
+  congo
+  croatia
+  cuba
+  curacao
+  democratic-republic-of-congo
+  denmark
+  djibouti
+  el-salvador
+  equatorial-guinea
+  eritrea
+  estonia
+  ethiopia
+  finland
+  gabon
+  georgia
+  greece
+  guatemala
+  guinea
+  guinea-bissau
+  haiti
+  honduras
+  hungary
+  iceland
+  iraq
+  jordan
+  kazakhstan
+  kyrgyzstan
+  latvia
+  libya
+  liechtenstein
+  lithuania
+  luxembourg
+  macedonia
+  mali
+  marshall-islands
+  mauritania
+  mexico
+  micronesia
+  moldova
+  montenegro
+  nepal
+  nicaragua
+  niger
+  norway
+  palau
+  panama
+  paraguay
+  romania
+  russia
+  rwanda
+  san-marino
+  sao-tome-and-principe
+  serbia
+  south-sudan
+  st-maarten
+  sudan
+  suriname
+  sweden
+  tajikistan
+  timor-leste
+  togo
+  tunisia
+  turkmenistan
+  uzbekistan
+  venezuela
+  western-sahara
 )
 
+# Exclude #country_without_consular_facilities? countries
+countries = countries -
+  SmartAnswer::Calculators::MarriageAbroadDataQuery::COUNTRIES_WITHOUT_CONSULAR_FACILITIES
+
+# Excluded countries ignored in outcome template
+countries = countries -
+  %w(cote-d-ivoire burundi)
```

Comment:
Minor: It wasn't immediately obvious to me why you needed to exclude these countries. It would be nice to explain a bit more either in the comment or in the commit note. I assume it's because of this condition `elsif %w(cote-d-ivoire burundi).exclude?(calculator.ceremony_country)`.